src/js/components/conversation.jsx

```diff
@@ -1,17 +1,30 @@
 import React, { Component } from 'react';
 import { findDOMNode } from 'react-dom';
 import { connect } from 'react-redux';
+import isMobile from 'ismobilejs';
+
 import { createMarkup } from 'utils/html';
 import { MessageComponent } from 'components/message.jsx';
 
 export class ConversationComponent extends Component {
-    constructor(...args) {
-        super(...args);
-        this.state = {
-            logoIsAnchored: true
-        };
-        this.scrollTimeouts = [];
-    }
+    state = {
+        logoIsAnchored: true
+    };
+
+    scrollTimeouts = [];
+
+    onTouchStart = () => {
+        const node = findDOMNode(this);
+        const top = node.scrollTop;
+        const totalScroll = node.scrollHeight;
+        const currentScroll = top + node.offsetHeight;
+
+        if (top === 0) {
+            node.scrollTop = 1;
```

Comment:
ehh, I should check if it's still working, but basically it's to prevent the overscroll. I think it's not working though.